scipy/sparse/tests/test_base.py

```diff
@@ -4498,9 +4499,17 @@ def cases_64bit():
                 msg = SKIP_TESTS.get(method_name)
                 if bool(msg):
                     marks += [pytest.mark.skip(reason=msg)]
-                for mname in ['skipif', 'skip', 'xfail', 'xslow']:
-                    if hasattr(method, mname):
-                        marks += [getattr(method, mname)]
+
+                if LooseVersion(pytest.__version__) >= LooseVersion("3.6.0"):
```

Comment:
the pytestmark attribute exists since pytest 3.2 https://github.com/pytest-dev/pytest/blob/master/CHANGELOG.rst#pytest-320-2017-07-30

originally the pin was for version 3.5+ so the variation shouldnt be needed

Reply:
The new code doesn't seem to work on pytest 3.5 --- not that it fails, but some skipif decorators don't get applied in the same way.